src/main/java/mate/academy/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -20,7 +20,7 @@ public CinemaHall add(CinemaHall cinemaHall) {
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
-            session.save(cinemaHall);
+            session.persist(cinemaHall);
```

Comment:
What is the reason for this change? And in the other files.

Reply:
Because the save method is deprecated and it is recommended to use the persist method instead.